### PR TITLE
Implement evm RPC into test framework

### PR DESCRIPTION
**Pull request by sieniven**
## Summary

- Modification of test framework to implement evm RPC
- Handling of EVM RPCs to the correct jsonrpc port on defid node

## Implications

- Storage
  - [ ] Database reindex required
  - [ ] Database reindex optional
  - [ ] Database reindex not required
  - [x] None

- Consensus
  - [ ] Network upgrade required
  - [ ] Includes backward compatible changes
  - [ ] Includes consensus workarounds
  - [ ] Includes consensus refactors
  - [x] None

### CODE REVIEW

**Comment by sieniven** [comment]
Commit #2ad8aca redirects the output of rust lib's env_logger to stdout instead of the default stderr.

**Comment by sieniven** [comment]
@Jouzo for sure. How should i showcase this? Currently the implementation is such that the TestNode class creates an instance of the HTTPConnection the the evm jrpc port, and stores a map of eth specific RPCs. When making RPCs to the node, we map to see if the call is eth-specific, and then pass it to the correct HTTPConnection instance accordingly.

File: lib/ain-grpc/src/lib.rs [hunk 1]
```diff
@@ -52,7 +52,7 @@ pub fn add_grpc_server(_runtime: &Runtime, _addr: &str) -> Result<(), Box<dyn Er
 
 pub fn init_runtime() {
     log::info!("Starting gRPC and JSON RPC servers");
-    LogBuilder::from_env(Env::default().default_filter_or(Level::Info.as_str())).init();
+    LogBuilder::from_env(Env::default().default_filter_or(Level::Info.as_str())).target(Target::Stdout).init();
```
Comment (prasannavl) [inline lib/ain-grpc/src/lib.rs:55]:
```suggestion
    LogBuilder::from_env(Env::default().default_filter_or(Level::Info.as_str()))
        .target(Target::Stdout)
        .init();
```

File: make.sh [hunk 2]
```diff
@@ -348,8 +348,10 @@ test_py() {
 
     _ensure_enter_dir "${release_target_dir}"
 
+    _fold_start "functional-tests"
```
Comment (prasannavl) [inline make.sh:351]:
```suggestion
```

File: make.sh [hunk 3]
```diff
@@ -348,8 +348,10 @@ test_py() {
...
     # shellcheck disable=SC2086
     ./test/functional/test_runner.py --tmpdirprefix "./test_runner/" --ansi "$@"
+    _fold_end
```
Comment (prasannavl) [inline make.sh:354]:
```suggestion
```

**Pull request merged**